libbeat/common/seccomp/policy_linux_arm.go

```diff
@@ -26,7 +26,6 @@ func init() {
 			{
 				Action: seccomp.ActionErrno,
 				Names: []string{
-					"execve",
```

Comment:
What are these changes? Why do we remove it from `arm` and add to `amd64`?

Reply:
ARM has got a block list for seccomp, so I remove `execve` from the block list. L27 states the action for those syscalls: `seccomp.ActionErrno`.